app/services/flow/flow_state_machine.rb

```diff
@@ -21,6 +21,11 @@ def show
     def update
       step = current_step
       result = flow.handle(step)
+
+      if params[:controller] == 'idv/doc_auth' && step == 'upload'
+        irs_attempts_api_tracker.document_upload_method_selected(upload_method: params[:type])
+      end
```

Comment:
would it make sense to move this into [`upload_step.rb`](https://github.com/18F/identity-idp/blob/main/app/services/idv/steps/upload_step.rb)?

Reply:
That is [where I had it before](https://github.com/18F/identity-idp/pull/6810/commits/5140c3b832d78f66f09b2cf0512281c38a3cba9f) - but I was not able to pass in the tracker correctly.  Do you know how I might keep it in the `upload_step` and pass in the `irs_attempts_api_tracker`?

Reply:
Thanks to some changes in https://github.com/18F/identity-idp/pull/6826/files#diff-8673af19f3f0a612a50730a5100e81f30c18ed195e33048af3699e042d069dbcR11, I think we can access it via `@flow.irs_attempts_api_tracker` in the steps now

Reply:
It worked!  Thanks!